src/JWT.js

```diff
@@ -67,10 +67,10 @@ export function decodeJWT (jwt) {
 *  })
 *
 *  @param    {Object}            payload            payload object
-*  @param    {Object}            [config]           an unsigned credential object
-*  @param    {String}            config.issuer      The DID of the issuer (signer) of JWT
-*  @param    {String}            config.alg         The JWT signing algorithm to use. Supports: [ES256K, ES256K-R], Defaults to: ES256K
-*  @param    {SimpleSigner}      config.signer      a signer, reference our SimpleSigner.js
+*  @param    {Object}            [options]           an unsigned credential object
```

Comment:
This was just moved over, but I find the description confusing, also generally what args are require here and which are not? if required maybe we don't have them in an options object. We can consider in another PR instead of this one, if you want to merge in, changes look good.